README.md

```diff
@@ -1,28 +1,26 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# CAPSTONE-CONFERENCE
 
-> Description the project.
+> In this capstone a page of a conference is presented, using creativity we include our own ideas for the conference to customize the project, we implemented all the concepts seen in the module, HTML, CSS, DOM, JS Objects, Linters. 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML & CSS
+- JS 
+
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link] https://diegoh40.github.io/capstone-conference-avengers/)
```

Comment:
- [ ] [Optional] You can consider adding ```(``` before the link to fix the markdown like so:
```[Live Demo Link](https://diegoh40.github.io/capstone-conference-avengers/)```